.github/BUILD.md

```diff
@@ -13,81 +13,109 @@ If the answer is yes, please read on. Otherwise, make sure to head on over [to t
+## Working with the Umbraco source code
 
 Did you read ["Are you sure"](#are-you-sure)?
 
 [More details about contributing to Umbraco and how to use the GitHub tooling can be found in our guide to contributing.][contribution guidelines]
 
 If you want to run a build without debugging, see [Building from source](#building-from-source) below. This runs the build in the same way it is run on our build servers.
 
-> [!NOTE]
-> The caching for the back office has been described as 'aggressive' so we often find it's best when making back office changes to [disable caching in the browser (check "Disable cache" on the "Network" tab of developer tools)][disable browser caching] to help you to see the changes you're making.
+If you've got this far and are keen to get stuck in helping us fix a bug or implement a feature, great! Please read on...
 
-#### Debugging with VS Code
+### Prerequisites
 
-In order to build the Umbraco source code locally with Visual Studio Code, first make sure you have the following installed.
+In order to work with the Umbraco source code locally, first make sure you have the following installed.
 
--   [Visual Studio Code](https://code.visualstudio.com/)
+-   Your favourite IDE: [Visual Studio 2022 v17+ with .NET 7+](https://visualstudio.microsoft.com/vs/), [Rider](https://www.jetbrains.com/rider/) or [Visual Studio Code](https://code.visualstudio.com/)
 -   [dotnet SDK v9+](https://dotnet.microsoft.com/en-us/download)
 -   [Node.js v20+](https://nodejs.org/en/download/)
 -   npm v10+ (installed with Node.js)
 -   [Git command line](https://git-scm.com/download/)
 
-Open the root folder of the repository in Visual Studio Code.
+### Familiarizing yourself with the code
+
+Umbraco is a .NET application using C#. The solution is broken down into multiple projects.  There are several class libraries. The `Umbraco.Web.UI` project is the main project that hosts the back office and login screen. This is the project you will want to run to see your changes.
+
+There are two web projects in the solution with client-side assets based on TypeScript, `Umbraco.Web.UI.Client` and `Umbraco.Web.UI.Login`.
+
+There are a few different ways to work locally when implementing features or fixing issues with the Umbraco CMS. Depending on whether you are working solely on the front-end, solely on the back-end, or somewhere in between, you may find different workflows work best for you.
+
+Here are some suggestions based on how we work on developing Umbraco at HQ.
 
-To build the front-end you'll need to open the command pallet (<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd>) and run `>Tasks: Run Task` followed by `Client Build`.
+### First checkout
 
-You can also run the tasks manually on the command line:
+When you first clone the source code, rebuild the whole solution via your IDE. You can then start the application via the IDE or the command line and should find everything across front and back-end is built and running.
```

Comment:
A rebuild is seldomly necessary, seldomly - only when something has gone horribly wrong. Just build.

Reply:
```suggestion
When you first clone the source code, build the whole solution via your IDE. You can then start the `Umbraco.Web.UI` project via the IDE or the command line and should find everything across front and back-end is built and running.
```